modin/core/dataframe/algebra/binary.py

```diff
@@ -56,6 +59,8 @@ def caller(query_compiler, other, broadcast=False, *args, **kwargs):
                 If `other` is a one-column query compiler, indicates whether it is a Series or not.
                 Frames and Series have to be processed differently, however we can't distinguish them
                 at the query compiler level, so this parameter is a hint that passed from a high level API.
+            dtypes : "copy" or None, default: None
+                Whether keep old dtypes or infer new dtypes from data.
```

Comment:
Parameter should be moved down after `args` parameter description, because `dtypes` is keyword parameter.

Reply:
fixed, thx